spec/requests/product_drives_requests_spec.rb

```diff
@@ -1,21 +1,97 @@
 require 'rails_helper'
 
-RSpec.describe "ProductDrives", type: :request do
-  let(:default_params) do
-    { organization_id: @organization.id.to_param }
```

Comment:
The original setup for this test was incorrect. Originally this organisation was not being used in the tests (instead, [due to this line](https://github.com/rubyforgood/human-essentials/blob/da164577b1dface4c82bbbb37b5799bd14591a7e/app/controllers/application_controller.rb#L17), a seeded/default organisation was being used instead).

I've therefore explicitly defined entities in the setup to make sure we use the defined orgainsation when running these tests

Reply:
Good catch! @shen-sat